languages/csharp/exercises/concept/nullability/.meta/design.md

```diff
@@ -0,0 +1,50 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to introduce the student to the concept of [Nullability in C#][null-keyword].
+
+## Learning objectives
+
+- Know of the existence of the `null` literal.
+- Know what a `NullReferenceException` is and when it is thrown.
+- Know how to compare a value to `null`.
+- Know the difference between value and reference types regarding nullability, especially pre C# 8.0.
+- Know how to define nullable reference and value types.
+- Know about the null-related operators (`!`, `?`, `??`).
+- Know about basic null checking by the compiler.
+
+## Out of scope
+
+- Nullable attributes.
+- In-depth discussion of null checking by the compiler.
+- Enabling C# 8 null checking.
+- Casting using the `as` or `is` operator or using pattern matching.
+
+## Concepts
```

Comment:
We've recently updated the [naming rules of concepts](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#naming-concepts), hence the concept naming changes I suggest in this review.